From 6f9955dfaccacc355cfff049fa6abdf280a0bdcc Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 23 Jan 2026 19:21:13 -0500
Subject: [PATCH] transform/livereloadinject/livereloadinject.go: Fix compile
 warnings

Fix this compile warning

/livereloadinject.go:76:63: (github.com/gohugoio/hugo/common/loggers.Logger).Warnf call has arguments but no formatting directives

Signed-off-by: Prarit Bhargava <prarit@redhat.com>
---
 transform/livereloadinject/livereloadinject.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/transform/livereloadinject/livereloadinject.go b/transform/livereloadinject/livereloadinject.go
index 55e69f912821..335233a7ecca 100644
--- a/transform/livereloadinject/livereloadinject.go
+++ b/transform/livereloadinject/livereloadinject.go
@@ -73,7 +73,7 @@ func New(baseURL *url.URL) transform.Transformer {
 		copy(c[idx+len(script):], b[idx:])
 
 		if _, err := ft.To().Write(c); err != nil {
-			loggers.Log().Warnf("Failed to inject LiveReload script:", err)
+			loggers.Log().Warnf("Failed to inject LiveReload script: %s", err)
 		}
 		return nil
 	}
-- 
2.52.0
